app/controllers/submissions_controller.rb

```diff
@@ -39,7 +39,7 @@ def new
     @participations = Participation.where({ user_id: current_user.id, active: true }).order('challenge_id ASC')
     # Inform the user of their houses score (if they are participating)
     @house_participations = current_user.house_participations.where("join_date >=  ?", Time.now.utc.at_beginning_of_month.to_date).first
-    @points = @house_participations ? @house_participations.score : 0
+    @time = @house_participations ? @house_participations.score : 0
```

Comment:
Not necessarily better, but there's also the option of `@house_participations&.score || 0`, if you prefer that syntax. (I think that'd work anyway, I haven't tried it.)